tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Azure/Functions/AzureFunctionsCommon.cs

```diff
@@ -34,9 +34,8 @@ internal static class AzureFunctionsCommon
 
             if (tracer.Settings.IsIntegrationEnabled(IntegrationId))
             {
-                // Ignoring null because guaranteed running in AAS
-                if (tracer.Settings.AzureAppServiceMetadata!.IsIsolatedFunctionsApp
-                    && tracer.InternalActiveScope is null)
+                if (tracer.Settings.AzureAppServiceMetadata is { IsIsolatedFunctionsApp: true }
```

Comment:
There's another `Ignoring null because guaranteed running in AAS` down in lines 114-115. Doesn't hurt to check for null there as well.